encoding/utils/codec/codec.go

```diff
@@ -0,0 +1,66 @@
+package codec
+
+import (
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+// ConvertByPaddingEmptyByte takes bytes and insert an empty at the front of every 31 byte
+// This ensure every 32 bytes are within the valid range of a field element for bn254 curve
```

Comment:
2^248 is only 2% of the field modulo, i suppose it doesn't pose a concern to use 2% of the space?

Reply:
It is a simple implementation, I think we can add more sophisticated methods later, or let others contribute.